tests/Cosmos/Protobuf/Article.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package blog;
+
+enum Type {
+  TYPE_UNSPECIFIED = 0;
+  IMAGES = 1;
+  NEWS = 2;
+};
+
+enum Review {
+  REVIEW_UNSPECIFIED = 0;
+  ACCEPTED = 1;
+  REJECTED = 2;
+};
+
+message Article {
```

Comment:
maybe we don't need this? we already have `MsgSend` test

Reply:
I did the test from the Cosmos SDK ADR; this has some test for some special cases.  It's true that rest of tests indirectly test serialization, but not necessarily all aspects.  It's important to have serialization identical to each byte.  I'd keep it.